multiplayer_server/Player.php

```diff
@@ -1312,8 +1312,8 @@ public function becomeServerOwner()
     public function becomeGuest()
     {
         $this->guest = true;
-        $this->group = 1;
-        $this->write('setGroup`1');
+        $this->group = 0;
+        $this->write('setGroup`0');
```

Comment:
You can take out lines 1315 and 1316 altogether.

Reply:
Oh, I thought it was needed

Reply:
It won't hurt anything, but it's already taken care of in the `process_register_login()` function in `process_fns.php`.